### ISSUE TITLE
Remove latex formatted text from doc string 

### ISSUE THREAD

**Issue by m-aguena** [LSSTDESC/CLMM#147]
There are some latex formatted equations in the doc strings of `modeling.py`, they will not be compiled correctly and break our tests. Must be fixed.

**Comment by aimalz** [comment]
Just commenting to say that #148 fixed the compilation error that broke the tests, but the [docs](https://clmm.readthedocs.io/en/master/api/clmm.modeling.html) aren't rendering right yet (at least for me), so the issue remains open.

**Comment by matthewkirby** [comment]
The docs can handle latex formatted equations! We just do not have our docstrings formatted correctly. Referenced from here: http://openalea.gforge.inria.fr/doc/openalea/doc/_build/html/source/sphinx/rest_syntax.html#maths-and-equations-with-latex

For inline math: ``Sentence with an equation like :math:`x = 2` ``

For more complicated math not inline:
```
.. math::

    n_{\mathrm{offset}} = \sum_{k=0}^{N-1} s_k n_k
```
The tabs are important.

**Comment by matthewkirby** [comment]
@m-aguena resolved this in PR #154 which is now merged with master!
